src/promptflow-tools/promptflow/tools/common.py

```diff
@@ -442,6 +444,55 @@ def render_jinja_template(prompt, trim_blocks=True, keep_trailing_newline=True,
         raise JinjaTemplateError(message=error_message) from e
 
 
+def render_jinja_template_wrapper(prompt, trim_blocks=True, keep_trailing_newline=True, **kwargs):
```

Comment:
the wrapper naming is ambigurious. escaping roles in prompt template inputs is a separate process from rendering.

Reply:
Agreed. 

Maybe a good process is: 

es_result= escape_roles()
render_result = render_jinja_template(es_result)
unescape_result = unescape_roles(render_result )